app/models/User.js

```diff
@@ -159,8 +165,17 @@ userSchema.methods.postDashbotOutgoing = function (gambitMessageType) {
   this.postDashbot('outgoing', gambitMessageType);
 };
 
-userSchema.methods.setLastOutboundTemplate = function (messageTemplate) {
-  this.last_outbound_template = messageTemplate;
+/**
+ * @param {string} messageTemplate
+ * @param {number} campaignId
+ */
+userSchema.methods.updateConversation = function (outboundTemplate, campaignId) {
```

Comment:
Not a blocker for merging, but I like what you did here: https://github.com/DoSomething/slothie/blob/master/app/models/User.js#L90-L120. (splitting up functions for statuses) Maybe we can bring that same logic here?